ctpolicy/loglist/loglist.go

```diff
@@ -0,0 +1,164 @@
+package loglist
+
+import (
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"math/rand"
+	"sync"
+	"time"
+
+	"github.com/letsencrypt/boulder/ctpolicy/ctconfig"
+	"github.com/letsencrypt/boulder/ctpolicy/loglist/generated"
+)
+
+// logList is a cached/memoized in-memory version of the autogenerated and
+// embedded JSON blob. We do this so we don't have to re-parse the whole blob
+// every time we want a new log list.
+var logList struct {
```

Comment:
I got mildly confused just now that there's a `var logList` and a `type LogList` but they are not closely related. Can we come up with a different name for one?